src/roadmap/data/systems.py

```diff
@@ -150,7 +152,7 @@
         major=8,
         minor=7,
         start_date=date(2022, 11, 9),
-        end_date=date(2023, 5, 16),
+        end_date=date(2023, 5, 31),
```

Comment:
just confirming, this change is expected?

Reply:
I came across those in the [Retired Life Cycle Dates](https://access.redhat.com/articles/4038291#red-hat-enterprise-linux-8-7) for RHEL 8.